CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+**Fixed bugs:**
```

Comment:
please do not update the changelog. we autogenerate it during the release.

Reply:
Apologies! It has been fixed.